BigQuery/1_demographic_and_stay_data.sql

```diff
@@ -28,9 +30,21 @@ JOIN
 USING
   (subject_id,
     hadm_id)
+LEFT JOIN 
+(
+    select subject_id, round(avg(valuenum), 2) as weight, max(valueuom) as weight_units 
```

Comment:
Why average weight, and selecting max units? Are there cases of differing units? In which case it does not makes sense to average the weight. It makes sense to convert wither to lbs. or kg. then average